dwave/cloud/api/client.py

```diff
@@ -345,3 +349,16 @@ class MetadataAPIClient(DWaveAPIClient):
     def __init__(self, **config):
         config.setdefault('endpoint', constants.DEFAULT_METADATA_API_ENDPOINT)
         super().__init__(**config)
+
+
+class LeapAPIClient(DWaveAPIClient):
+    """Client for D-Wave's Leap API."""
+
+    @classmethod
+    def _add_auth(cls, session, config):
+        if config['token']:
+            session.headers.update({'Authorization': f"Bearer {config['token']}"})
```

Comment:
Bike shedding, I generally find `.update(Authorization=...)` a bit easier to read, and it also avoids the creation of an intermediate dictionary.

Reply:
I generally prefer kwargs as well, but HTTP headers often contain hyphens (like `Accept-Language`), so I prefer string keys in that case.